include/unix/osd.h

```diff
@@ -37,6 +37,8 @@
 #include <unistd.h>
 #include <errno.h>
 
+#define FI_WAIT_MUTEX_CRITSEC FI_WAIT_MUTEX_COND
```

Comment:
there are about 20 locations where this macro used... and here is decision about bit confusing & #ifdef spaghetti
what do you prefer? :)